src/EditorFeatures/CSharpTest/Extensions/TelemetryExtensionTests.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using Microsoft.CodeAnalysis.Editor.Shared.Extensions;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.Extensions
+{
+    public class TelemetryExtensionTests
+    {
+        [Fact]
+        public void TestConstantTelemetryId()
+        {
+            var expected = Guid.Parse("00000000-0000-0000-c4c5-914100000000");
```

Comment:
Could we get the new Hash to be the last 8 digits of the Guid?

Reply:
The new hash algorithm just doesn't take up that much since it hashes to a 32 bit integer, which only takes up 4 bytes of hex. I don't think we'll have a problem with it, even though it's not aesthetically as pleasing.